presto-main/src/test/java/com/facebook/presto/operator/exchange/TestLocalExchange.java

```diff
@@ -239,13 +239,11 @@ public void testPartition()
 
         assertSource(sourceA, 1);
         assertSource(sourceB, 1);
-        assertExchangeTotalBufferedBytes(exchange, 1);
```

Comment:
Can we use `assertTrue(exchange.getBufferedBytes() > retainedSizeOfPages(1))`?  I would assume a splitting a page into multiple parts is always larger than the input size?

If not, that is ok.